styles.css

```diff
@@ -0,0 +1,158 @@
+body {
+    width: 1024px;
+    margin: 0 auto;
+    background-image: url(http://ges-design.ru/library/collage/background/me/background-15-21.jpg);
+    font-family: 'K2D', sans-serif;
+}
+
+header {
+    position: fixed;
+    width: 1024px;
+    z-index: 1;
+}
+
+div.pass {
```

Comment:
`html/css`: `.pass`. `.skills`, `.sum`, `.foot` are elements with common mistakes. First of all, you could use full names (footer instead of foot). Second, they have common layout logic, relative position, margin from top element, paddings etc. You could add such common properties into common class, e,g. `.container` and add it to each of them. Inside specific classes like `.pass` you could add specific properties like gradients.